components/Auction/Auction.module.css

```diff
@@ -1,167 +1,162 @@
+  cursor: pointer;
 }
 
-.auctionContainer {
+.auctionButtonContentWrapper {
   display: flex;
-  width: 70%;
-  background-color: rgb(245, 235, 235);
-  justify-content: space-around;
+  justify-content: center;
   align-items: center;
-  border: 1px solid #f4f4f4;
-  border-radius: 20px;
-  box-shadow: 0 0 15px - 7px rgba(0, 0, 0, 0.65);
-  margin-bottom: 20px;
-  padding: 50px 0;
+  gap: 4px;
 }
 
-.auctionSeller {
+.mainContainer {
   display: flex;
   flex-direction: column;
-  align-items: center;
+  gap: 2rem;
+  margin-bottom: 6.25rem;
 }
 
-.auctionSeller {
-  width: 100px;
+.autionsPageTitle {
+  font-size: 1.5rem;
+  font-weight: 600;
+  line-height: 2rem;
+  color: #352e5b;
 }
 
-.itemInfo {
-  display: flex;
-  justify-content: center;
+.auctionWrapper {
+  display: grid;
+  grid-template-columns: minmax(17.5rem, 40.6rem);
+  gap: 1.5rem;
 }
 
-.gemImage {
-  margin-top: 20px;
+.auctionContainer {
+  background: #ffffff;
+  border: 1px solid #ffffff;
+  border-radius: 6px;
+  padding: 1.875rem 2rem;
+  display: grid;
+  grid-template-columns: 5rem 1fr auto;
+  gap: 2rem;
 }
 
-.currentBidPrice {
-  font-weight: 400;
+.auctionContainer:hover {
+  box-shadow: 0px 3px 6px rgba(75, 81, 91, 0.15),
+    0px 1px 3px rgba(0, 0, 0, 0.15);
 }
 
-.bidOptions {
+.auctionSeller {
+  width: 5rem;
   display: flex;
   flex-direction: column;
+  align-items: center;
 }
 
-.inputBid {
-  border: 0.5px solid #ccc;
-  border-radius: 5px;
-  padding: 10px;
-  margin-bottom: 15px;
-  font-size: 1rem;
-  width: 150px;
-}
-
-.bidBtn {
-  height: 50px;
-  width: 150px;
-  margin-top: 10px;
-  color: white;
-  border-color: rgba(27, 31, 35, 0.15);
-  border-radius: 6px;
-  background-color: #42a742;
+.profilePhoto {
+  border-radius: 50%;
+  width: 100%;
 }
 
-.bidBtn:hover {
-  height: 50px;
-  box-shadow: 0 0 15px - 7px rgba(0, 0, 0, 0.65);
-  cursor: pointer;
+.auctionStatsBidderWrapper {
+  display: flex;
+  flex-direction: column;
+  gap: 2.5rem;
 }
 
-.bidders {
+.itemInfo {
   display: flex;
-  flex-wrap: wrap;
-  width: 25%;
+  align-items: center;
+  margin-bottom: 6px;
+  gap: 4px;
 }
 
-.profilePhoto {
-  border-radius: 50%;
-  width: 100%;
+.currentStatus {
+  display: flex;
+  gap: 4px;
 }
 
-.biddersImg {
-  margin-left: 10px;
+.itemInfo,
+.currentStatus {
+  font: 1.125rem bold;
+  color: #373b53;
+  flex-wrap: wrap;
 }
 
-.biddersImg[data-columns='2'] {
-  width: calc(160px / 2);
+.gemImage {
+  margin-top: 1.125rem;
 }
 
-.biddersImg[data-columns='3'] {
-  width: calc(160px / 3);
+.currentBidPrice {
+  font: 1.125rem bold;
+  color: #e30062;
 }
 
-.biddersImg[data-columns='4'] {
-  width: calc(160px / 4);
+.bidders {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, 50px);
+  gap: 4px;
 }
 
-.biddersImg[data-columns='5'] {
-  width: calc(160px / 5);
+.bidOptions {
+  display: flex;
+  flex-direction: column;
 }
 
-.biddersImg[data-columns='6'] {
-  width: calc(160px / 6);
+.inputBid {
+  border: 0.5px solid #ccc;
```

Comment:
can you please use hex value, it will be consistent

Reply:
Done.